src/ClimaAtmos/Dycore/drivers/isentropic_vortex.jl

```diff
@@ -67,18 +68,19 @@ function isentropicvortex(t, x...)
   u = uinf - λ*(1//2)*exp(1-rsq)*yp/π
   v = vinf + λ*(1//2)*exp(1-rsq)*xp/π
   w = zero(DFloat)
-
   ρ = (Tinf - ((γ-1)*λ^2*exp(2*(1-rsq))/(γ*16*π*π)))^(1/(γ-1))
+  
   p = ρ^γ
   U = ρ*u
   V = ρ*v
   W = ρ*w
   E = p/(γ-1) + (1//2)*ρ*(u^2 + v^2 + w^2)
```

Comment:
```suggestion
E = p/(γ-1) + ρ * internal_energy(0) + (1//2)*ρ*(u^2 + v^2 + w^2)
```

Reply:
@jkozdon Thanks  - you're right about this - perhaps it raises a general question of "reference states" in the general case against which parameters are normalised.

Reply:
possibly. Hopefully some of these issues will be clearer when the new framework is up.